WeakAurasTemplates/TriggerTemplatesData.lua

```diff
@@ -999,20 +1035,24 @@ else
           { spell = 6673, type = "ability"}, -- Battle Shout
           { spell = 12975, type = "ability", buff = true}, -- Last Stand
           { spell = 18499, type = "ability", buff = true}, -- Berserker Rage
-          { spell = 20243, type = "ability", requiresTarget = true, talent = {16, 17}}, -- Devastate
+          { spell = 20243, type = "ability", requiresTarget = true, talent = {1, 2}}, -- Devastate
+          { spell = 22409, type = "ability", requiresTarget = true}, -- Storm Bolt
```

Comment:
![image](https://user-images.githubusercontent.com/189656/95002086-9c62f700-05d0-11eb-8ad0-d3c38904f644.png)
```suggestion
          { spell = 107570, type = "ability", requiresTarget = true}, -- Storm Bolt
```